http-clients/src/test/java/com/palantir/remoting/http/GuavaOptionalAwareContractTest.java

```diff
@@ -114,7 +114,7 @@ public void testStringQuery() throws Exception {
     public void testAbsentHeader() throws Exception {
         proxy.header(Optional.<String>absent(), "str2");
         RecordedRequest takeRequest = server.takeRequest();
-        assertThat(takeRequest.getHeader("opt"), is("{opt}"));
+        assertThat(takeRequest.getHeader("opt"), is(""));
```

Comment:
yeah I'd have liked to keep this out of the header params all-together but that requires feign changes